docs/source/private-data-arch.rst

```diff
@@ -373,7 +373,7 @@ applied to the channel’s state database to enable JSON content queries, by
 packaging indexes in a ``META-INF/statedb/couchdb/indexes`` directory at chaincode
 installation time.  Similarly, indexes can also be applied to private data
 collections that are explicitly defined, by packaging indexes in a ``META-INF/statedb/couchdb/collections/<collection_name>/indexes``
-directory. An example index is available `here <https://github.com/hyperledger/fabric-samples/blob/{BRANCH}/chaincode/marbles02_private/go/META-INF/statedb/couchdb/collections/collectionMarbles/indexes/indexOwner.json>`_.
+directory. An example index is available `here <https://github.com/hyperledger/fabric-samples/blob/{BRANCH}/asset-transfer-ledger-queries/chaincode-go/META-INF/statedb/couchdb/indexes/indexOwner.json>`_.
```

Comment:
The link you've provided refers to a general State Index example. For this case, we should use an index example for Private Data Collection instead:
```suggestion
directory. An example index is available `here <https://github.com/hyperledger/fabric-samples/blob/{BRANCH}/asset-transfer-private-data/chaincode-go/META-INF/statedb/couchdb/collections/assetCollection/indexes/indexOwner.json>`_.
```

Reply:
We also have not been creating fabric-samples branches to match fabric branches. Therefore the `BRANCH` variable should switch to `main`. This could be done in this PR, or it could be done in a separate PR to correct each of the occurrences.

Reply:
Hello @satota2 and @denyeart , Thank you for suggestions. I have updated the json file link to represent a private data collection state index. I have updated link to use `main` as branch name instead of `BRANCH` variable. I will raise a dedicated PR by fixing all similar occurrences.

Reply:
Thank you!